src/main/java/org/eolang/jeo/matchers/SameXml.java

```diff
@@ -78,19 +74,37 @@ public void describeTo(final Description description) {
     }
 
     /**
-     * Remove 'line' attributes from the XML document.
-     * @param original Original XML document.
-     * @return XML document without 'line' attributes.
+     * The same XML, but without lines.
+     * @since 0.6
      */
-    private static XML withoutLines(final XML original) {
-        try {
-            return new XMLDocument(
-                new Xembler(
-                    new Directives().xpath(".//o[@line]/@line").remove()
-                ).apply(original.node())
-            );
-        } catch (final ImpossibleModificationException exception) {
-            throw new IllegalStateException("Failed to remove 'line' attributes", exception);
+    public static final class WithoutLines implements Scalar<XML> {
```

Comment:
@h1alexbel What do you think if we move this class upper to `matchers` package? I mean, we can make this class independent (not inner class).

Reply:
Good idea. I think we can make it `public final`, in case someone want to use that